README.md

```diff
@@ -4,10 +4,19 @@
   <a href="https://github.com/actions/setup-ruby"><img alt="GitHub Actions status" src="https://github.com/actions/setup-ruby/workflows/Main%20workflow/badge.svg"></a>
 </p>
 
-This action sets up a ruby environment for use in actions by:
+This action sets up Ruby for use in actions by adding to the `PATH` a version of Ruby already existing in the toolcache.
```

Comment:
I think this should just say: **This action only supports versions installed on the actions virtual environments**.  The tool cache is an internal implementation detail.

Reply:
OK